src/Robot.cpp

```diff
@@ -1,87 +1,79 @@
 #include <memory>
+#include <signal.h>
+#include <iostream>
 
 #include <Commands/Command.h>
 #include <Commands/Scheduler.h>
-#include <IterativeRobot.h>
 #include <LiveWindow/LiveWindow.h>
 #include <SmartDashboard/SendableChooser.h>
 #include <SmartDashboard/SmartDashboard.h>
 
-#include "Commands/ExampleCommand.h"
 #include "CommandBase.h"
 
-class Robot: public IterativeRobot {
+#include "CardinalDash/IterativeRobot.h"
+#include "CardinalDash/Server.h"
+
+class Robot: public CardinalDash::IterativeRobot {
 public:
+    static void term(int signum)
+    {
+        CardinalDash::Server::Term();
+    }
+
 	void RobotInit() override {
-		// chooser.AddDefault("Default Auto", new ExampleCommand());
-		// chooser.AddObject("My Auto", new MyAutoCommand());
-		// SmartDashboard::PutData("Auto Modes", &chooser);
+        struct sigaction action;
+        memset(&action, 0, sizeof(struct sigaction));
+        action.sa_handler = (void(*)(int))term;
+        sigaction(SIGTERM, &action, NULL);
+
+        IterativeRobot::RobotInit();
+
+        CommandBase::Init();
 	}
 
-	/**
-	 * This function is called once each time the robot enters Disabled mode.
-	 * You can use it to reset any subsystem information you want to clear when
-	 * the robot is disabled.
-	 */
-	void DisabledInit() override {
+    void RobotPeriodic() override {
+        IterativeRobot::RobotPeriodic();
```

Comment:
wanna use super/inherited? http://stackoverflow.com/questions/180601/using-super-in-c